### PR TITLE
fix service manager

### PR DESCRIPTION
**Pull request by berndtj**
Service manager was broken due to the changes with regards to passing
org IDs around.

Extened the timeouts to try and address the e2e test failures.

### CODE REVIEW

**Comment by kars7e** [comment]
Looks like there is an issue with one of the mocks, unit tests don't pass

**Comment by berndtj** [comment]
Ugh... I need more logs.  The tests work for me.

**Comment by berndtj** [comment]
Ugh... confused how the last patch passed e2e with the above failures... should be a relatively easy fix though.

File: pkg/client/secrets.go [hunk 1]
```diff
@@ -53,6 +54,7 @@ func (c *DefaultSecretsClient) CreateSecret(ctx context.Context, organizationID
 		XDispatchOrg: c.getOrgID(organizationID),
 		Secret:       secret,
 	}
+	log.Infof("creating secret %s/%s", organizationID, *secret.Name)
```
Comment (kars7e) [inline pkg/client/secrets.go]: Some debug leftovers?

Comment (berndtj) [reply pkg/client/secrets.go]: I decided to leave it in... I can remove

Comment (berndtj) [reply pkg/client/secrets.go]: done

**Pull request merged**